afs-security-analysis-local/src/main/java/com/powsybl/afs/security/local/LocalSecurityAnalysisRunningService.java

```diff
@@ -32,14 +33,6 @@
  */
 public class LocalSecurityAnalysisRunningService implements SecurityAnalysisRunningService {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(LocalSecurityAnalysisRunningService.class);
-
-    private final Supplier<SecurityAnalysisFactory> factorySupplier;
```

Comment:
See comment in the extension class: we should have a `SecurityAnalysisRunner` here as a replacement for the factory, in order to define what implementation of security analysis we want to use